src/mono/netcore/sample/wasm/Directory.Build.props

```diff
@@ -0,0 +1,20 @@
+<Project>
+  <PropertyGroup>
+    <Configuration>Release</Configuration>
+    <OutputType>Exe</OutputType>
+
+    <RunAOTCompilation Condition="'$(RunAOTCompilation)' == ''">false</RunAOTCompilation>
+    <PublishTrimmed>true</PublishTrimmed>
+    <TrimMode>link</TrimMode>
+    <RunAnalyzers>false</RunAnalyzers>
+  </PropertyGroup>
+
+  <Import Project="..\Directory.Build.props"/>
+
+  <PropertyGroup>
+    <OutputPath>bin</OutputPath>
```

Comment:
Why the separation between this PropertyGroup with the one above?

Reply:
Just to work with how the Directory.Build.props sets up things, and depends on some properties being set.